test/spdmlib-test/src/responder_tests/examples/certificate_case_positive.yaml

```diff
@@ -0,0 +1,31 @@
+---
+input:
```

Comment:
What if the test certificate is changed?

Reply:
Yes, it's hard to change if hardcode certificate bytes. Therefore I added a new parser to generate expected output. Please take a look.